code/iomethods.py

```diff
@@ -276,10 +234,10 @@ def read_data(dirName, varName, latName, lonName, filelist=None):
         alllonsraw[alllonsraw > 180] = alllonsraw[alllonsraw > 180] - 360.  # convert to -180,180 if necessary
 
         #get the lat/lon info data (different resolution)
-        latminNETCDF = utils.find_nearest(alllatsraw, float(LATMIN))
-        latmaxNETCDF = utils.find_nearest(alllatsraw, float(LATMAX))
-        lonminNETCDF = utils.find_nearest(alllonsraw, float(LONMIN))
-        lonmaxNETCDF = utils.find_nearest(alllonsraw, float(LONMAX))
+        latminNETCDF = utils.find_nearest(alllatsraw, float(userVariables.LATMIN))
```

Comment:
We should cast these values to a float when the userVariables object is defined.  I know that when reading from JSON it all comes in as text, so it would be nice to have the variables typed correctly so calling functions don't have to re-type them.